src/main/java/com/google/sps/EventFormInput.java

```diff
@@ -0,0 +1,59 @@
+package com.google.sps;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+
+
+@WebServlet("/get-info")
+public class EventFormInput extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+     // Sanitize user input to remove HTML tags and JavaScript.
+    String Type = Jsoup.clean(request.getParameter("type"), Whitelist.none());
+    String Date = Jsoup.clean(request.getParameter("date"), Whitelist.none());
+    double Price = Double.valueOf(request.getParameter("price"));
+    String State = Jsoup.clean(request.getParameter("state"), Whitelist.none());
+    String Address = Jsoup.clean(request.getParameter("address"), Whitelist.none());
+    String Zip = Jsoup.clean(request.getParameter("zip"), Whitelist.none());
+
+    //connverts Date to Epoch Time//
```

Comment:
nit: spelling on "converts"

also add a space after the initial comment tag, and no need for an ending one

`// Converts Date to Epoch Time`